Gruntfile.js

```diff
@@ -282,6 +282,8 @@ module.exports = function(grunt) {
       options: {
         urls: ["http://127.0.0.1:9999/test/tests.html"],
         testname: "Plottable Sauce Unit Tests",
+        pollInterval: 5000,
+        statusCheckAttempts: 60,
```

Comment:
Let's see if this leads to less failures.  I don't think the increases in time outweigh the pain of the tests flaking.